playbooks/hc-ansible-deployment/gluster_inventory.yml

```diff
@@ -305,6 +305,9 @@ hc_nodes:
 
   # Common configurations
   vars:
+    # In case of IPv6 based deployment "gluster_features_enable_ipv6" needs to be enabled, like:
+    # gluster_features_enable_ipv6: true
```

Comment:
Will the user uncomment this to enable ipv6?

Reply:
Corrected with next commit